spec/PushController.spec.js

```diff
@@ -1200,10 +1200,12 @@ describe('PushController', () => {
         isLocalTime: false
       })).toBe('2007-04-05T14:30:00.000Z', 'Timezone offset');
 
+      const noTimezone = new Date('2017-09-06T17:14:01.048')
```

Comment:
@marvelm could you have a look please? As you were originally building that feature.

Thanks!